packages/react/src/components/TimePicker/next/TimePicker-test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { default as TimePicker } from './TimePicker';
+
+import { render, cleanup, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+describe('TimePicker', () => {
+  describe('input', () => {
+    afterEach(cleanup);
+
+    it('renders as expected', () => {
+      render(<TimePicker id="time-picker"></TimePicker>);
```

Comment:
```suggestion
      render(<TimePicker id="time-picker" />);
```

If a component has no children we can use `<ComponentName />` as a shorthand